### PR TITLE
[Py] Added gelu operator to Python API.

### PR DESCRIPTION
**Pull request by Ewa21**

### CODE REVIEW

File: python/ngraph/ops.py [hunk 1]
```diff
@@ -527,6 +527,24 @@ def convert(node, new_type, name=None):  # type: (Node, NumericType, str) -> Nod
     return Convert(node, new_element_type)
 
 
+@nameable_op
+def gelu(node, name=None):  # type: (NodeInput, str) -> Node
+    r"""Perform Gaussian Error Linear Unit operation element-wise on data from input node.
```
Comment (tsocha) [inline python/ngraph/ops.py]: Was this 'r' character intended?

Comment (Ewa21) [reply python/ngraph/ops.py]: Yes, it was. `r` is a prefix for Python raw string. It treats backslash as a literal character, not as an escape character.

File: python/pyngraph/ops/fused/elu.hpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//*****************************************************************************
```
Comment (postrational) [inline python/pyngraph/ops/fused/elu.hpp]: Why is this file in this PR?

File: python/pyngraph/ops/fused/gelu.cpp [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+#include "ngraph/op/fused/gelu.hpp"
+#include "pyngraph/ops/fused/gelu.hpp"
+
+namespace py = pybind11;
+
+void regclass_pyngraph_op_Gelu(py::module m)
+{
+    py::class_<ngraph::op::Gelu, std::shared_ptr<ngraph::op::Gelu>, ngraph::op::Op> gelu(m, "Gelu");
+    gelu.doc() = "ngraph.impl.op.Elu wraps ngraph::op::Gelu";
```
Comment (postrational) [inline python/pyngraph/ops/fused/gelu.cpp]: `ngraph.impl.op.Elu`

File: python/ngraph/ops.py [hunk 4]
```diff
@@ -527,6 +527,24 @@ def convert(node, new_type, name=None):  # type: (Node, NumericType, str) -> Nod
...
+
+    Computes Gaussian error linear:
```
Comment (postrational) [inline python/ngraph/ops.py]: `Gaussian error linear` -> `GELU function`

**Comment by postrational** [comment]
Please make sure the math formula renders correctly in the docs.

**Pull request merged**